### PR TITLE
feat: Account for functions with containers for analytics

### PR DESCRIPTION
**Pull request by pgrzesik**
Add detection of functions defined with images + first installation timestamp

note: it should wait before corresponding backend services are deployed

### CODE REVIEW

File: lib/utils/analytics/generatePayload.js [hunk 1]
```diff
@@ -100,8 +100,13 @@ module.exports = async (serverless) => {
       plugins: serviceConfig.plugins ? serviceConfig.plugins.modules || serviceConfig.plugins : [],
       functions: Object.values(serviceConfig.functions).map((functionConfig) => {
         const functionEvents = functionConfig.events || [];
+        const functionRuntime = (() => {
+          if (functionConfig.image) return 'container_image';
```
Comment (medikoo) [inline lib/utils/analytics/generatePayload.js:104]: Maybe we can make more special name, e.g. `$containerimage` ? (I've added `$` to indicate it's not real runtime and removed space to maintain same convention with other runtimes, e.g. it's `nodejs` not `node_js`), WDYT

**Review by medikoo — COMMENTED** [review]
@pgrzesik looks great. I've just proposed to give more _special_ name to this special case.

File: lib/utils/analytics/generatePayload.js (hunk 1 above)
Comment (pgrzesik) [reply lib/utils/analytics/generatePayload.js]: I like that idea :+1: Changed

**Review by medikoo — APPROVED** [review]
Looks great 👍

**Pull request merged**